ner_v2/detectors/textual/elastic_search.py

```diff
@@ -142,6 +143,12 @@ def get_multi_entity_results(self, entities, texts, fuzziness_threshold=1,
         try:
             response = self._run_es_search(self._default_connection, **kwargs)
             results = _parse_multi_entity_es_results(response.get("responses"))
+        except es_exceptions.NotFoundError as e:
```

Comment:
F841 local variable 'e' is assigned to but never used

Reply:
resolved in latest commit